host.go

```diff
@@ -243,8 +243,8 @@ func (h *Host) Serve() {
 	h.listener.Serve()
 }
 
-func (h *Host) completeName(partial string) string {
-	names := h.NamesPrefix(partial)
+func (h *Host) completeName(partial string, current_user *message.User) string {
```

Comment:
Go uses camelCase instead of snake_case for variables.

```suggestion
func (h *Host) completeName(partial string, currentUser *message.User) string {
```